tests/transforms/test_intensity.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import kornia.augmentation as K
+import pytest
+import torch
+from torch import Tensor
+
+from torchgeo.transforms import Sentinel1ChangeMap, ToDecibelScale
+
+
+@pytest.fixture
+def sample() -> dict[str, Tensor]:
```

Comment:
Someday (not necessarily in this PR) we should probably add a `tests/transforms/conftest.py` to define all of these shared fixtures.